### PR TITLE
Alternative for resizable dock

### PR DESCRIPTION
**Pull request by vlebert**
I had some bugs with Resize Dock with splitter](./library/ui/resize_dock_with_splitter) so I made an alternative using [interact.js](https://interactjs.io/).

### CODE REVIEW

File: library/ui/resize_dock_2/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Resize dock 2
+
+I had some bugs with Resize Dock with splitter](./library/ui/resize_dock_with_splitter) so I made an alternative using [interact.js](https://interactjs.io/).
```
Comment (Gustry) [inline library/ui/resize_dock_2/README.md:3]:
```suggestion
An alternative implementation of the other script [resize dock with splitter](./resize_dock_with_splitter), using [interact.js](https://interactjs.io/).
```

**Comment by nworr** [comment]
Is the orginal demo.gif revelant for this script, can it be added  ?

**Comment by vlebert** [comment]
I can. I made an other gif but was too heavy

**Comment by vlebert** [comment]
just this actual script only makes the left dock resizable

**Comment by Gustry** [comment]
For the GIF, is-it a different UI ? Or can you make it lite ?
If not, just a preview like : 

```md
![demo](../foo/demo.gif)
```

I will let my colleague for the JS part

**Review by nworr — APPROVED** [review]

**Review by Gustry — APPROVED** [review]
Thanks.

Let's merge

**Pull request merged**